src/pudl/metadata/codes.py

```diff
@@ -352,6 +352,95 @@
+                ),
+                (
+                    "BS",
+                    "baghouse_shake_deflate",
+                    "Baghouse (fabric filter), shake and deflate",
+                ),
+                ("CD", "circulating_dry_scrubber", "Circulating dry scrubber"),
+                (
+                    "DSI",
+                    "dry_sorbent_injection",
+                    "Dry sorbent (powder) injection type",
+                ),
+                (
+                    "EC",
+                    "electrostatic_cold_conditioned",
+                    "Electrostatic precipitator, cold side, with flue gas conditioning",
+                ),
+                (
+                    "EH",
+                    "electrostatic_hot_conditioned",
+                    "Electrostatic precipitator, hot side, with flue gas conditioning",
+                ),
+                (
+                    "EK",
+                    "electrostatic_cold_unconditioned",
+                    "Electrostatic precipitator, cold side, without flue gas conditioning",
+                ),
+                (
+                    "EW",
+                    "electrostatic_hot_unconditioned",
+                    "Electrostatic precipitator, hot side, without flue gas conditioning",
+                ),
+                (
+                    "JB",
+                    "wet_scrubber_jet_bubbling",
+                    "Jet bubbling reactor (wet) scrubber",
+                ),
+                ("LIJ", "lime_injection", "Lime injection"),
+                ("LNB", "low_nox_burner", "Low NOx burner"),
+                (
+                    "MA",
+                    "wet_scrubber_mechanical",
+                    "Mechanically aided type (wet) scrubber",
+                ),
+                ("MC", "multiple_cyclone", "Multiple cyclone"),
+                ("OT", "other", "Other"),
+                ("PA", "wet_scrubber_packed", "Packed type (wet) scrubber"),
+                ("SC", "single_cyclone", "Single cyclone"),
+                (
+                    "SD",
+                    "spray_dryer",
+                    "Spray dryer type / dry FGD / semi-dry FGD",
+                ),
+                (
+                    "SN",
+                    "selective_noncatalytic_reduction",
+                    "Selective noncatalytic reduction",
+                ),
+                ("SP", "wet_scrubber_spray", "Spray type (wet) scrubber"),
+                (
+                    "SR",
+                    "selective_catalytic_reduction",
+                    "Selective catalytic reduction",
+                ),
+                ("TR", "wet_scrubber_tray", "Tray type (wet) scrubber"),
+                ("VE", "wet_scrubber_venturi", "Venturi type (wet) scrubber."),
+            ],
+        ).convert_dtypes(),
+        "code_fixes": {
+            "SR-2": "SR",
+            "sn": "SN",
+            "LN": "LNB",
+        },
+        "ignored_codes": ["HRSG1", "HRSG2", "FGD", "OV"],
```

Comment:
Why are these ones ignored? Are they not specified in the EIA documentation?

Reply:
FGD is a pre-2013 code for "flue gas desulfurization" (see the `mercury_compliance_strategies_eia` coding table). 
OV is "overfire air" (see the `nox_compliance_strategies_eia` coding table).

Reply:
I think there's a whole bunch of notes about this in the original PR for those encoders #2319. Some codes get more or less specific over time, and sometimes where the details get reported changes over time. It'd be wise for us to do some bigger reconciling across all of these columns and I think this is even identified as a potential step in #2338.

You've double-checked that the layout codes haven't changed over time (esp pre-2013), correct?

^ If so, I think leaving this as-is is fine but we should note explicitly in #2672 that we should revisit codes that have gotten dropped.

Reply:
Sigh.. no I was lazy and just looked at the 2021 data. I'll check out the older ones and see how they map to each other.

Reply:
Okay, I've searched back through several different Layout spreadsheets (including pre-2013) to see if FGD or OV were ever valid codes for the environmental control equipment, and they never seem to have been.  In addition, FGD is showing up in 2018 and OV is showing up in 2013, and those codes definitely weren't valid equipment types in those years, so I think they are data entry errors and can't be cleanly mapped to any particular equipment type, so they should probably be dropped.

Unfortunately I don't think we'll be able to fill in either of these codes via #2616 either, since the OV codes show up only in the first year of data for their emissions unit, and the FGD code shows up sandwiched between two other valid codes which are different from each other in earlier / later years.

I've updated #2672 to remind us to look at the dropped codes.

Reply:
Great, I'm ok to drop them then. Thanks for looking into it.